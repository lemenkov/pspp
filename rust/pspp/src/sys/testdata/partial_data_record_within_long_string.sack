# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 2; 0; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Numeric variables.
2; 14; 0; 0; 0x010e00 *2; s8 "STR14";
2; -1; 0; 0; 0; 0; s8 "";

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# Data.
999; 0;
s14 "one data item";
s8 "partial";
