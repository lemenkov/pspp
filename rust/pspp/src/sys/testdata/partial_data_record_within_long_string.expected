╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │None                    │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─╮
│Variables│1│
╰─────────┴─╯

                                               Variables
╭─────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│     │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├─────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│str14│       1│     │Nominal          │Input│   14│Left     │A14         │A14         │              │
╰─────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯

I/O error (failed to fill whole buffer)

╭────┬──────────────╮
│Case│     str14    │
├────┼──────────────┤
│1   │one data item │
╰────┴──────────────╯
